respa_exchange/downloader.py

```diff
@@ -30,6 +36,11 @@ def _populate_reservation(reservation, ex_resource, item_props):
     """
     reservation.begin = item_props["start"]
     reservation.end = item_props["end"]
+    subject = item_props["subject"]
+    if subject is None:
+        with push_scope() as scope:
+            scope.level = 'warning'
+            capture_message('Calendar item has empty subject')
     reservation.event_subject = item_props["subject"]
```

Comment:
We will still get IntegrityError when saving the reservation with null subject(?)